README.rst

```diff
@@ -39,14 +29,14 @@ The other known implementations are:
 * Go: `dripline-go <https://github.com/project8/dripline-go>`_
 * LabView: `dripline-labview <https://github.com/project8/dripline-labview>`_
 
-.. _getting-help:
 
 Getting help
 ++++++++++++
-Code documentation is produced using sphinx via reStructuredText which is distributed with the source. A [rendered](http://www.project8.org/dripline) version is hosted using github pages and the project8.org domain.
-Questions related to a particular implementation may be posted to the associated issue tracker; generic questions in the issue tracker for this repo.
 
-.. _other-notes:
+Dripline documentation can be [here](https://driplineorg.github.io).
```

Comment:
this link formatting is broken when viewed on GitHub? (also throughout the following paragraph)

Reply:
That's a md link, not an rst link